uint16 x
uint16 y
uint16 camera_id
uint8 point_id
time stamp